src/styles/utils/variables.scss

```
@@ -0,0 +1,3 @@
+$main-text-color: #060B35;
+$second-text-color: #616070;
+$bg-color: #00ACDC;
```

Let's use more descriptive name for this variable
```suggestion
$bg-color: #00ACDC;
```